v1/counterpartymetadata/counterpartymetadata.gunk

```diff
@@ -0,0 +1,2138 @@
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type CreatePrivateAliasRequest struct {
+	// PrivateAlias is the private_alias metadata made on a other_account.
+	PrivateAlias PublicAlias `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type UpdatePrivateAliasRequest struct {
+	// PrivateAlias is the private_alias metadata made on a other_account.
+	PrivateAlias PublicAlias `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type DeletePrivateAliasRequest struct {
+	// BankID is the bank identifier.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetPrivateAliasRequest struct {
+	// BankID is the bank identifier.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetPrivateAliasesRequest struct {
+	// BankID is the bank identifier.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetPrivateAliasesResponse struct {
+	// PrivateAliases is the list of private_aliases.
+	PrivateAliases []PublicAlias `pb:"1" json:"aliases"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type GetOtherAccountMetadataRequest struct {
+	// BankID is the bank identifier.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetOtherAccountMetadataResponse struct {
+	PublicAlias       string            `pb:"1" json:"public_alias"`
+	PrivateAlias      string            `pb:"2" json:"private_alias"`
```

Comment:
May be we should match `public_alias` and `private_alias` json tag all over the structs, few places it was mentioned just as `alias`